tutormfe/templates/mfe/build/mfe/env/production

```diff
@@ -22,5 +22,6 @@ SEGMENT_KEY=''
 SITE_NAME='{{ PLATFORM_NAME|replace("'", "'\\''") }}'
 STUDIO_BASE_URL='{{ "https" if ENABLE_HTTPS else "http" }}://{{ CMS_HOST }}'
 USER_INFO_COOKIE_NAME='user-info'
+ENABLE_LEARNER_RECORD_MFE='true'
```

Comment:
This setting is not defined here: https://edx.readthedocs.io/projects/edx-developer-docs/en/latest/micro_frontends/index.html#required-environment-variables
This means that it's a setting that is specific to the learning MFE, right? If yes, I suggest to add it only to the learning MFE configuration. This is possible by adding `"env": {"production": {"ENABLE_LEARNER_RECORD_MFE": "true"}}` to the "LEARNING_MFE_APP" config entry in plugin.py.

Source:
https://github.com/overhangio/tutor-mfe/blob/5807603a9893d355357d4379ef1b4e634ecc2fd4/tutormfe/templates/mfe/build/mfe/Dockerfile#L59
https://github.com/overhangio/tutor-mfe/blob/5807603a9893d355357d4379ef1b4e634ecc2fd4/tutormfe/plugin.py#L32

Reply:
Yup you are totally right... just re pushed